### ISSUE TITLE
WMS GetFeatureInfo (geojson) on raster group not working

### ISSUE THREAD

**Issue by swalterfub** [mapserver/mapserver#5806]
I have two grouped raster layers sitting next to each other and I want to query the raster values using GetFeatureInfo and GeoJson. When I query on one of the layer names, I get a good result. But when I query on the grouped layers, I get no result and an error message

> [Fri May 17 09:08:07 2019].242137 msOGRWriteFromQuery(): General error message. OGR OGR_DS_CreateLayer failed for layer 'HRSC-MC11W' with driver 'GEOJSON'.

instead.
When I change the grouping so that it contains only one layer, the query works as well.
Here is my OUTPUTFORMAT definition:
 OUTPUTFORMAT
  NAME "geojson"
  DRIVER "OGR/GEOJSON"
  MIMETYPE "application/json"
  FORMATOPTION "STORAGE=stream"
  FORMATOPTION "FORM=SIMPLE"
END

**Comment by boazhuisman** [comment]
We are experiencing the same problem.

**Comment by jratike80** [comment]
Does if work if you query the layers of the layer group as a list `&QUERY_LAYERS=layer_1,layer_2,layer_3`?

**Comment by boazhuisman** [comment]
No, sadly that does not work. Each layer has a different geometry, very much like this thread:

**Comment by jmckenna** [comment]
When you report a problem here, please always take the time to create a small test case (mapfile, data, commands.txt) and attach it to your ticket as a .zip.
